### ISSUE TITLE
require path breaking intellisense for javascript files

### ISSUE THREAD

**Issue by ctrlaltcookie** [Microsoft/vscode#37126]
I have a problem which is caused by using `./` in the path of a require, for example: `const AVariable = require('./index');` which breaks intellisense and most navigation functionality. VScode on mac.

- VSCode Version: 1.17.2 (1.17.2)
- OS Version: Mac Sierra 10.12.6 

Steps to Reproduce:

1. Add any require anywhere in a file with a path that navigates up a folder using `./`
2. Observe all navigation and intellisense no longer working.

Reproduces without extensions: Yes

**Comment by mjbvz** [comment]
Can you please test this with [VS Code insiders](https://code.visualstudio.com/insiders/). The insiders builds bundle a new version of TypeScript which fixes a few import bugs around this path intellisense

**Comment by ctrlaltcookie** [comment]
I'll test it early tomorrow morning and get back to you. I'm UK time.
Thanks.

**Comment by ctrlaltcookie** [comment]
Yep the insiders build fixes this, thanks a bunch you can close this, sorry to bother :)
